### ISSUE TITLE
Odd behaviour, only half the functions working...

### ISSUE THREAD

**Issue by Osiris-Ra** [bitfocus/companion-module-magichome-bulb#1]
Experiencing a strange behaviour where only some of the functions work for some of my light and everything works on the others. Same behaviour on Companion 2.4 and 3.0
Got 4 of the same led controllers and 2 of them all the functions work but the other two only some functions work. 
I can power toggle all 4, set patterns and modes, but 2 of the controllers not able to change the colour with any mode not change the brightness but the other two work fine.

**Comment by josephdadams** [comment]
Maybe they are different models. I have found some of mine behave differently also. Maybe share firmware versions or models of the bulbs so we can look into it together

**Comment by Osiris-Ra** [comment]
Yup defiantly looks like a firmware issue, The top and center ones are the ones I can fully control so that firmware is good and the left and rights ones can turn on and off and change patterns but not colour change or brightness so they must be using different commands which makes sense with their different firmware. 
V1   06.v1.11.7376-a    -   center
V1   06.v1.11.7376-a    -   top
V3   06.v3.31.7786-a    -   left
V10 33.v10.48.7658-a  -   right

**Comment by Osiris-Ra** [comment]
Any update?

**Comment by josephdadams** [comment]
I thought we determined it was a problem with your firmware

**Comment by Osiris-Ra** [comment]
ah ok, so there no way to be able to control the lights then?
